app/assets/stylesheets/partials/wice_grid.scss

```diff
@@ -126,7 +113,7 @@
   ul {margin-left: 0 }
 
   a.wice-grid-delete-query .delete-icon{
-    background: transparent url(/assets/icons/grid/delete.png) no-repeat;
+    background: transparent asset-url('/assets/icons/grid/delete.png') no-repeat;
```

Comment:
Color literals like `transparent` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets<br>Prefer double-quoted strings